pkg/core/resources/apis/meshexternalservice/api/v1alpha1/zz_generated.resource.go

```diff
@@ -18,14 +22,33 @@ import (
 var rawSchema []byte
 
 func init() {
-	var schema spec.Schema
+	var structuralSchema *schema.Structural
+	var schemaObject spec.Schema
+	var v1JsonSchemaProps *apiextensionsv1.JSONSchemaProps
 	if rawSchema != nil {
-		if err := yaml.Unmarshal(rawSchema, &schema); err != nil {
+		if err := yaml.Unmarshal(rawSchema, &schemaObject); err != nil {
+			panic(err)
+		}
+		rawJson, err := yaml.YAMLToJSON(rawSchema)
```

Comment:
we need JSON because the unmarshaling functions are only defined for JSON and CBOR but not for YAML https://github.com/kubernetes/kubernetes/blob/c81431de59a3bf516489317433a165b050322339/staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1/marshal.go#L125